plugins/basestate/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+project(basestate)
```

```suggestion
project(basestate)
```

I think we can remove this.

---

We can remove this since it is not referenced anywhere right?